seqr/views/apis/variant_search_api_tests.py

```diff
@@ -274,6 +284,8 @@ def test_query_variants(self, mock_get_variants, mock_get_gene_counts, mock_erro
         response_json = response.json()
         self.assertSetEqual(set(response_json.keys()), set(EXPECTED_SEARCH_RESPONSE.keys()))
         self.assertDictEqual(response_json, EXPECTED_SEARCH_RESPONSE)
+        self.assertListEqual(response_json['phenotypeGeneScores']['I000001_na19675']['ENSG00000268903']['lirical'],
+                             EXPECTED_LIRICAL_DATA)
```

Comment:
just put `EXPECTED_LIRICAL_DATA` into `EXPECTED_SEARCH_RESPONSE`, there is no reason to test it separatley